examples/DrawingInTheAir/DrawingInTheAir_MotionActivated

```diff
@@ -0,0 +1,290 @@
+/*
+ * This example demonstrates using the pattern matching engine (CuriePME)
+ * to classify streams of accelerometer data from CurieIMU.
+ *
+ * First, the sketch will prompt you to draw some letters in the air (just
```

Comment:
This whole description is just a straight copy paste, which is mostly fine. However it *would* be nice if you had some specific information too, like for example, what's the difference between this sketch and the plain DrawingInTheAir sketch? put that extra information here, in the description